browserdebuggertools/chrome/interface.py

```diff
@@ -107,17 +107,29 @@ def execute_javascript(self, script):
         return result.get("value")
 
     def get_url(self):
-        return self.execute_javascript("document.URL")
+        # type: () -> str
+        """
+        Consider enabling the Page domain to increase performance.
```

Comment:
Or else the socket handler will spam them with a 'domain not enabled' error message every time we try to get the url (because it calls the PageLoadEventHandler, which calls check_page_load, which will try to get the Page events from the socket handler and catch the DomainNotEnabledError).

Is that what you want?